vendor/github.com/go-jose/go-jose/v3/crypter.go

```diff
@@ -524,18 +526,18 @@ func (obj JSONWebEncryption) DecryptMulti(decryptionKey interface{}) (int, Heade
 		}
 	}
 
-	if plaintext == nil || err != nil {
+	if plaintext == nil {
 		return -1, Header{}, nil, ErrCryptoFailure
 	}
 
 	// The "zip" header parameter may only be present in the protected header.
 	if comp := obj.protected.getCompression(); comp != "" {
-		plaintext, err = decompress(comp, plaintext)
+		plaintext, _ = decompress(comp, plaintext)
```

Comment:
This was also part of the lint cleanup in https://github.com/go-jose/go-jose/commit/7d3b437c. I'm not sure it's correct! It seems like the fix should have been to check the error, not ignore it.